client/extensions/woocommerce/app/order/index.js

```diff
@@ -46,10 +66,12 @@ class Order extends Component {
 		];
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site.slug } breadcrumbs={ breadcrumbs } />
+				<ActionHeader breadcrumbs={ breadcrumbs }>
+					<Button primary onClick={ this.saveOrder } disabled={ isSaving }>{ translate( 'Save Order' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Save New Order' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-orders-action-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).